fossology/fossology
# chore(debian): Fix lintian erros and warnings

## Description

Fix some errors and warnings reported by lintian.

### Changes

1. Start executable files in `/usr/share` with a `#!`
    - This effects PHP files from agents:
        1. decider
        1. deciderjob
        1. readmeoss
        1. reportImport
        1. reuser
        1. unifiedreport
        1. spdx2
1. Ignore insane long lines on minified JavaScript files.
1. Fix typos in:
    1. `debian/changelog`
    1. `debian/control`
    1. `debian/fossology-common.README.Debian`
1. Update Wiki link in `install/INSTALL`
1. Update debhelper compatibility level to match Debian Jessie and Ubuntu Xenial Xerus.
1. Split lines > 80 character in `debian/control`.
1. Add aditional meta data to `debian/upstream/metadata` and `debian/watch`.
1. Replace `-s/--same-arch` to `-a/--arch` in `debian/rules`
    - `-s` yields warning: `-s/--same-arch is deprecated; please use -a/--arch instead. This feature will be removed in compat 12.`

## How to test

1. Build a source package for fossology
    1. Run `make tar-release` and change name of the resulting tar to `fossology_3.5.0.orig.tar.gz`.
    1. Copy all files to a new folder without the `.git` folder.
    1. Run `debuild -us -uc -S`.
        1. Check the lintian logs.
1. Build a binary package for fossology
    1. Build `fossology-3.5.0.orig.tar.gz` as mentioned above.
    2. Run `debuild -us -uc -b`.
        1. Check the lintian logs.

    **OR**

    1. Take the source packages built above.
    2. Extract the source using `dpkg-source -x .../path/to/fossology_3.5.0-1.dsc`.
    3. Run `debuild -us -uc -b`.
        1. Check the lintian logs.

# Note
This PR also remove dependency of `fossology-db` from `fossology-web` package.

Closes #69
